rules/postgres.yml

```diff
@@ -0,0 +1,32 @@
+groups:
+  - name: PostgreSQL
+    rules:
+      - alert: PostgreSQL Down
+        expr: pg_up == 0
+        for: 5m
+        labels:
+          severity: error
+        annotations:
+          summary: "PostgreSQL is down (instance {{ $labels.instance }})"
+      - alert: PostgreSQL Too Many Connections
+        expr: sum by (datname) (pg_stat_activity_count{datname!~"template.*|postgres"}) > 100
```

Comment:
is 100 good number?

Reply:
I still don't know. Please suggest me, or tell me how to get a good number for this.

Reply:
for now we can go with this number